### PR TITLE
Updated to fix Issue 307

### PR DESCRIPTION
**Pull request by mattl91**

### CODE REVIEW

File: HarmonyCore.CliTool/TUI/Views/EditSettingView.cs [hunk 1]
```diff
@@ -177,13 +203,18 @@ public EditSettingView(string title, IHasNavigationResult navigationObject) : ba
             }
         }
 
+        private void Rg_SelectedItemChanged(SelectedItemChangedArgs obj)
```
Comment (hippiehunter) [inline HarmonyCore.CliTool/TUI/Views/EditSettingView.cs:206]: this doesnt look like its used anywhere

File: HarmonyCore.CliTool/TUI/Views/EditSettingView.cs [hunk 2]
```diff
@@ -101,16 +102,41 @@ public EditSettingView(string title, IHasNavigationResult navigationObject) : ba
...
+                        Y = 11,
+                        Width = Dim.Fill() - 4,
+                        Height = 3,
+                        ReadOnly = true,
+                        WordWrap = true,
+                    };
+                    Add(lbl, rg, affordance);
+                    ok.Enabled = false;
                     rg.SetFocus();
+                    rg.SelectedItemChanged += item =>
```
Comment (hippiehunter) [inline HarmonyCore.CliTool/TUI/Views/EditSettingView.cs:124]: is it possible to de-select an item or is it once selected, always selected?

Comment (mattl91) [reply HarmonyCore.CliTool/TUI/Views/EditSettingView.cs:124]: Once you select an item, you can deselect it by selecting another item (another radio button). Same as before. If you want to deselect whatever's selected without selecting something else, you Cancel.

File: HarmonyCore.CliTool/TUI/Views/EditSettingView.cs (hunk 1 above)
Comment (mattl91) [reply HarmonyCore.CliTool/TUI/Views/EditSettingView.cs]: It isn't. Fixed (i.e., removed).

**Pull request merged**